doc/user/playground.md

```diff
@@ -22,23 +22,48 @@ The command line arguments for playground state:
 ```bash
 Usage:
   tiup playground [version] [flags]
+  tiup [command]
+
+Available Commands:
+  completion  generate the autocompletion script for the specified shell
+  display
+  help        Help about any command
+  scale-in
+  scale-out
 
 Flags:
-      --db int                   TiDB instance number (default 1)
+      --db int                   TiDB instance number
+      --db.Host host             Playground TiDB host. If not provided, TiDB will still use host flag as its host
```

Comment:
This should be in lower case, it was a bug fixed in #1635

Reply:
```suggestion
      --db.host host             Playground TiDB host. If not provided, TiDB will still use host flag as its host
```

Reply:
This isn't yet fixed in v1.7.0 / ce8eb0a645cc3ead96a44d67b1ecd5034d112cf0

It is fixed in the master branch ( 9edf64ea0c1917a8aab2c355448471aa5e1419ee )